pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -726,6 +730,29 @@ def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
                     RemoveRedundancies(),
                 ]
             )
+        else:
+            passlist.extend(
+                [
+                    RemoveBarriers(),
+                    AutoRebase({OpType.CX, OpType.Rz, OpType.H}),
+                    GreedyPauliSimp(
+                        allow_zzphase=True,
+                        only_reduce=True,
+                        thread_timeout=timeout,
+                        trials=10,
```

Comment:
When `GreedyPauliSimp` returns false, will the user know the cause, so they can, for example, try increasing the timeout?

Reply:
They won't & I can't think of a good way of sharing this information in the code. Might just have to make it clear in the documentation?

Reply:
I see. Can we print out some information in the C++ code? If you think that's possible, we should create an issue for this.